cmake/FindSpatiaLite.cmake

```diff
@@ -17,7 +17,12 @@ if(PkgConfig_FOUND)
 endif()
 
 if(PC_SPATIALITE_FOUND)
-  add_library(spatialite::spatialite ALIAS PkgConfig::PC_SPATIALITE)
+  set(SPATIALITE_INCLUDE_DIR "${PC_SPATIALITE_INCLUDE_DIRS}" CACHE STRING "")
```

Comment:
The ALIAS we use is requiring the CMake 3.18+ syntax mentioned at https://cmake.org/cmake/help/latest/command/add_library.html#alias-libraries?
Could you keep the existing line but protected by a ``if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.18)`` and put your workaround in the else() clause, so we can later cleanup things when we remove CMake < 3.18 compatibility ?